### DISCUSSION TITLE
Contrast theme not working properly

### DISCUSSION THREAD

**Discussion by a2bhijeet** [cefsharp/CefSharp#4709] [General]
We are facing issues with contrast themes while using cefsharp. Please can anyone provide ways we can debug the problem or any documents that will help with contrast in CEFSharp

### CefSharp Version
120.2.70

### Operating System
Windows 11

### Architecture
x64

### .Net Version
.Net 4.7.2

### Implementation
WinForms

**Comment by amaitland** [comment]
What is exactly is `contract theme`? Do you mean Dark mode?

What is the issue exactly? Did you check the log file for errors?

Does the problem reproduce with https://github.com/cefsharp/CefSharp.MinimalExample
